README.md

```diff
@@ -14,29 +14,23 @@ Our chat room is `#skonfig:matrix.org` (you need [Matrix client](https://matrix.
 
 ## Getting Started
 
-There will be no versioned releases fow now - everything is expected to be used from the repositories.
-
-We are currently targeting (`cdist`) power users, so expect some rough edges.
-
 ### Step 1: Install `skonfig`
 
 ```sh
-git clone https://github.com/skonfig/skonfig
-ln -s "$PWD/skonfig/bin/skonfig" "$HOME/.local/bin/skonfig"
+pip3 install skonfig@git+https://github.com/skonfig/skonfig
 ```
 
-### Step 2: Create configuration directory and initial manifest
+### Step 2: Clone base and extra sets repositories
 
 ```sh
-mkdir -p "$HOME/.skonfig/manifest"
-cp "$PWD/skonfig/docs/examples/init-manifest" "$HOME/.skonfig/manifest/init"
+mkdir -p ~/.skonfig/set
+git clone https://github.com/skonfig/base ~/.skonfig/set/base
+git clone https://github.com/skonfig/extra ~/.skonfig/set/extra
 ```
 
-### Step 3: Clone base and extra sets repositories
+### Step 3: Create configuration directory and initial manifest
 
 ```sh
-mkdir "$HOME/.skonfig/set"
-cd "$HOME/.skonfig/set"
-git clone https://github.com/skonfig/base
-git clone https://github.com/skonfig/extra
+mkdir -p ~/.skonfig/manifest
+cp "$(pip show skonfig | sed -n -e 's;^Location: \(.*\)/lib.*$;\1;p')/share/doc/skonfig/examples/init-manifest" ~/.skonfig/manifest/init
```

Comment:
This is ugly, but I have no better suggestion. Ignore this comment.

Reply:
Yes it is. I'm not a big fan of it either but couldn't find a better solution that handles all the different locations where pip might install skonfig.

Maybe it would make sense to create a little helper utility to create a "getting started" `.skonfig` dir?
It could also take care of cloning sets.

Reply:
Sure, optional util, why not, but let's keep it like this here and revisit this in separate issue/PR.

Reply:
#37